build/toolchain/win/BUILD.gn

```diff
@@ -77,7 +77,11 @@ template("msvc_toolchain") {
       # TODO(brettw) enable this when GN support in the binary has been rolled.
       #precompiled_header_type = "msvc"
       pdbname = "{{target_out_dir}}/{{target_output_name}}_c.pdb"
-      command = "ninja -t msvc -e $env -- $cl /nologo /showIncludes /FC @$rspfile /c {{source}} /Fo{{output}} /Fd$pdbname"
+      flags = ""
+      if (is_clang && invoker.current_cpu == "x86") {
+        flags = "-m32"
```

Comment:
Chromium's approach is to add this [in build/config/win](https://source.chromium.org/chromium/chromium/src/+/master:build/config/win/BUILD.gn;l=111?q=m32%20file:gn$&ss=chromium&originalUrl=https:%2F%2Fcs.chromium.org%2F); I have a local change where I was experimenting with Windows arm64 support where I ended up needing to add that block as well. I can move this whenever I land the initial arm plumbing.

Reply:
Good thing about current setup is that it is aligned with dart https://github.com/dart-lang/sdk/blob/master/build/toolchain/win/BUILD.gn.
What are the advantages of Chromiums approach?

Reply:
> is that it is aligned with dart

Since I've routinely needed to re-sync and/or cherry-pick things from Chromium when adding new functionality (like arm64 support) to the Flutter build system, I don't know that "aligned with Dart" is more of an advantage than "aligned with Chromium".

> What are the advantages of Chromiums approach?

I don't have an authoritative answer, but personally this looks a lot to me like partially re-inventing `cflags`, which is already a built-in concept.